ballast-fairness-survey/ballast-fairness-survey.md

```diff
@@ -105,13 +105,14 @@ We will now ask your opinion on the main solutions proposed.
 - [ ] Liability/Certification: Could affect EN/CCC certification status; organizers mandating them may face liability in the event of an incident (should be answered in [FAQ](https://gagglereport.org/ballast/equalizers)).  
 - [ ] Safety/Installation: Pilots might install them incorrectly, creating a safety risk.  
 - [ ] Unfairness: Unfairly "punishes" heavier pilots who have no choice but to fly large wings.  
-- [ ] Ineffective: Won't truly equalize performance (e.g., handling, climb).  
-- [ ] Ineffective: Won't stop light pilots from ballasting up to M-sizes for better "comfort" or "handling."  
+- [ ] Ineffective: Won't truly equalize performance (e.g., handling, climb), leaving a performance gap.
```

Comment:
I don't like the "gap" word like it would still be not enough but actually it can be different and not adequate, as it's various performance differences between sizes and a simple drag tool cannot address all of them adequately. It's hard to verify if the compensation is too little or too much.